platform/admin/core/admin-core-appservice/src/test/java/org/codice/ddf/admin/application/service/impl/ApplicationServiceBeanTest.java

```diff
@@ -60,13 +57,11 @@
 import org.codice.ddf.admin.core.api.ConfigurationAdmin;
 import org.codice.ddf.admin.core.api.Service;
 import org.codice.ddf.admin.core.impl.ServiceImpl;
-import org.hamcrest.CustomMatcher;
+import org.codice.ddf.test.common.mockito.StackCaptor;
```

Comment:
❓ Do we still need that?

Reply:
At this point I'm leaning towards _no_, but I thought I'd leave it around until we get everybody's feedback.